{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2019-14856",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/bugzilla.redhat.com\/show_bug.cgi?id=CVE-2019-14856",
                    "name": "https:\/\/bugzilla.redhat.com\/show_bug.cgi?id=CVE-2019-14856",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "ansible before versions 2.8.6, 2.7.14, 2.6.20 is vulnerable to a None"
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2019-11-26T14:15Z",
    "lastModifiedDate": "2019-11-26T18:30Z"
}